scripts/verify_included_modules.py

```diff
@@ -131,7 +130,12 @@ def get_public_modules(path, base_package=None):
                 if base_package is not None:
                     rel_path = os.path.join(base_package, rel_path)
                 # Turn into a Python module rather than a file path.
-                result.append(rel_path.replace(os.path.sep, '.'))
+                rel_path = rel_path.replace(os.path.sep, '.')
+                if mod_name == '__init__':
+                    # len('.__init__') == 9
+                    result.append(rel_path[:-9])
```

Comment:
Why not compute it?  E.g.:

```python
                    result.append(rel_path[:-len('.__init__')])
```

Or store it in a local variable, if microoptimization makes your boat float. :)